backend/IRC.ml

```diff
@@ -459,44 +449,7 @@ let initialNodePartition g =
           DLinkNode.insert n g.simplifyWorklist
     | Colored -> ()
     | _ -> assert false in
-  Hashtbl.iter part_node g.varTable
-
-
-(* Check invariants *)
```

Comment:
These functions (up to and including `checkInvariants`) are useful to debug the register allocator.  I'd like to keep them but it's OK to comment them out.